README.adoc

```diff
@@ -865,76 +865,76 @@ And you just repeat these truth over and over and over:
 * FLG is good
 * xref:flg-religion[Alternate dimensions]
 * FLG is good
-* xref:flg-medication[Maybe FLG will make you healthier, but no promises, but it will make you a better person, and that's why you should do it, but if you really really believe, you get healthier]
+* xref:flg-medication[Maybe FLG will make you healthier, but no promises, but it will make you a better person, and that’s why you should do it, but, if you really really believe, you get healthier]
```

Comment:
I think let's keep away from typographic apostrophe and use just the ASCII one. Realistically I will never enter characters that are not available easily on my keyboard. There may be an Asciidoctor way to do it, but I don't know it.